addons/godot-firebase/firestore/firestore_collection.gd

```diff
@@ -15,31 +16,31 @@ signal update_document(doc)
 signal delete_document()
 signal error(code,status,message)
 
-var base_url : String
-var extended_url : String
+var _base_url : String
+var _extended_url : String
 var config : Dictionary
 var auth : Dictionary
 var collection_name : String
 
-var pusher : HTTPRequest
-
-
-const event_tag : String = "event: "
-const data_tag : String = "data: "
-const put_tag : String = "put"
-const patch_tag : String = "patch"
-const separator : String = "/"
-const json_list_tag : String = ".json"
-const query_tag : String = "?"
-const auth_tag : String = "auth="
-const authorization_header : String = "Authorization: Bearer "
-const auth_variable_begin : String = "["
-const auth_variable_end : String = "]"
-const filter_tag : String = "&"
-const escaped_quote : String = "\""
-const equal_tag : String = "="
-const key_filter_tag : String = "$key"
-const documentId_tag : String = "documentId="
+var _pusher : HTTPRequest
+
+
+const _event_tag : String = "event: "
+const _data_tag : String = "data: "
+const _put_tag : String = "put"
+const _patch_tag : String = "patch"
+const _separator : String = "/"
+const _json_list_tag : String = ".json"
+const _query_tag : String = "?"
+const _auth_tag : String = "auth="
+const _authorization_header : String = "Authorization: Bearer "
```

Comment:
This is something which should be more easily accessed. I'm thinking in firebase.gd itself, since we use it across technologies.